src/main/java/core/basesyntax/suppliers/Color.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.suppliers;
+
+public enum Color {
+    BLUE("blue"),
+    YELLOW("yellow"),
+    GREEN("green"),
+    WHITE("white"),
+    BLACK("black"),
+    RED("red");
+
+    private String color;
+
+    Color(String color) {

```

Comment:
you've missed `private` modifier
```suggestion
    private Color(String color) {

```

Reply:
Private Color()? Maybe Public Color()?

Reply:
actually no, because enum is a constant, you don't create them via constructor:
```
Color color = Color.WHITE; // correct, here your private constructor will be called with color parameter equal to "white"
Color color = new Color("white") // incorrect, since enum is no more a constant as you can pass any value into the constructor
```

Reply:
> actually no, because enum is a constant, you don't create them via constructor:
> 
> ```
> Color color = Color.WHITE; // correct, here your private constructor will be called with color parameter equal to "white"
> Color color = new Color("white") // incorrect, since enum is no more a constant as you can pass any value into the constructor
> ```

IDEA doesn't like that idea, it highlights key private in gray.